.circleci/ds005_outputs.txt

```diff
@@ -19,13 +19,21 @@ smriprep/sub-01/anat/sub-01_dseg.nii.gz
 smriprep/sub-01/anat/sub-01_from-fsnative_to-T1w_mode-image_xfm.txt
 smriprep/sub-01/anat/sub-01_from-T1w_to-fsnative_mode-image_xfm.txt
 smriprep/sub-01/anat/sub-01_hemi-L_inflated.surf.gii
+smriprep/sub-01/anat/sub-01_hemi-L_inflated.surf.json
```

Comment:
All these contain:

```JSON
{
  "EulerNumber": 2
}
```

Reply:
Wrong in what way?

Reply:
AFAIK, the Euler number will always be 2 on surfaces that have been inflated or smoothed, as each hemisphere will be 1.0 (no topological errors).

Reply:
are you saying the FreeSurfer algorithm is wrong? Because the interface just extracts the value printed to console..

Reply:
The Euler number is a straightforward value: Vertices + Faces - Edges. This is 2 for anything topologically equivalent to a sphere (no holes, no handles, no pinches), so it's just a way of detecting defects (determining in advance whether the surface can be smoothly deformed to a sphere).

Reply:
https://en.wikipedia.org/wiki/Euler_characteristic

Reply:
> are you saying the FreeSurfer algorithm is wrong?

I meant it doesn't make sense to calculate the Euler number in surfaces other than the raw (i.e., no smoothing or inflation) white and the pial. Sorry the comment was very unspecific and confusing.